### PR TITLE
eplot: 2.07 -> 2.09

### LINKED ISSUE #126925: eplot: out of date, 2.07 -> 2.09
###### Checklist

- [x] Checked the [nixpkgs master branch](https://github.com/NixOS/nixpkgs)

- [x] Checked the [nixpkgs pull requests](https://github.com/NixOS/nixpkgs/pulls)

###### Project name
`nix search` name:

current version: 2.07
desired version: 2.09

###### Notify maintainers

@bjornfor 

maintainers:

###### Note for maintainers
There are newer versions mentioned in the code, but I don't see an official release:

https://github.com/chriswolfvision/eplot/blob/master/eplot#L27

The latest version for example supports `--ylog` for logarithmic scales.

Please tag this issue in your PR.

Thank you!

### PR DESCRIPTION
**Pull request by SCOTT-HAMILTON**
###### Motivation for this change
Closes https://github.com/NixOS/nixpkgs/issues/126925

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

## Pinging maintainer
@bjornfor

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 126937` at d3441eae run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>eplot</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`installPhase` should probably contain `runHook preInstall` and `runHook postInstall`.

Near pkgs/tools/graphics/eplot/default.nix:22:3:
```
   |
22 |   installPhase = ''
   |   ^
```
</li>
  </ul>
</details>

**Review by bjornfor — CHANGES_REQUESTED** [review]
The sources must be updated.

I see that I made this package with unstable URLs (because there was nothing else). Perhaps we can fetch something from github now? And ping upstream about tagging.

File: pkgs/tools/graphics/eplot/default.nix [hunk 1]
```diff
@@ -1,7 +1,7 @@
 { lib, stdenv, fetchurl, gnuplot, ruby }:
 
 stdenv.mkDerivation {
-  name = "eplot-2.07";
+  name = "eplot-2.09";
```
Comment (SuperSandro2000) [inline pkgs/tools/graphics/eplot/default.nix:4]:
```suggestion
  pname = "eplot";
  version = "2.09";
```

File: pkgs/tools/graphics/eplot/default.nix [hunk 2]
```diff
@@ -42,6 +42,6 @@ stdenv.mkDerivation {
     homepage = "http://liris.cnrs.fr/christian.wolf/software/eplot/";
```
Comment (SuperSandro2000) [inline pkgs/tools/graphics/eplot/default.nix:42]:
```suggestion
    homepage = "https://perso.liris.cnrs.fr/christian.wolf/software/eplot/";
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by bjornfor** [comment]
Sources were not updated.

**Pull request merged**